h/views/activity.py

```diff
@@ -162,6 +162,27 @@ def user_annotation_count(aggregation, userid):
         return result
 
     @view_config(request_method='POST',
+                 request_param='group_join')
```

Comment:
Totally agree, I think it'll also make it much  more obvious what's happening by having two specific routes for joining and leaving.